di-ipv-core-stub/src/main/java/uk/gov/di/ipv/stub/core/CoreStub.java

```diff
@@ -28,6 +29,10 @@ public CoreStub() throws Exception {
 
     private void initRoutes() throws Exception {
         CoreStubHandler coreStubHandler = new CoreStubHandler(new HandlerHelper(getEcPrivateKey()));
+        BasicAuthHandler basicAuthHandler = new BasicAuthHandler();
+        if (CoreStubConfig.ENABLE_BASIC_AUTH) {
+            Spark.before(basicAuthHandler.authFilter);
```

Comment:
We can call this method CoreStubConfig.getUserAuth() here before add this filter. Please check to BasicAuthHandler comment. So, we can throw exception and logging.